VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,245 @@
+
+### Can a validator run away with its delegator’s ATOM?
+
+By delegating to a validator, a user delegates staking power. The more staking power a validator has, the more weight it has in the consensus and governance processes. However, this does not mean that the validator has custody of its delegator’s ATOMs. *By no means a validator can run away with its delegator’s funds*. 
+
+Even though delegator’s funds cannot be stolen by their validators, they are still liable if their validators misbehave. In such case, each delegator’s stake will be partially slashed in proportion of the total stake of the validator.
+
+### What is the incentive to stake?
+
+Each member of a validator’s staking pool earns different types of rewards:
+* **Block rewards:** Native tokens of applications run by validators (e.g. ATOMs) are inflated to produce block rewards. 
+* **Transaction fees:** The Hub maintains a whitelist of token that are accepted as fee payment. These fees are part of the total reward received by validators and delegators.
+
+These rewards are divided among validator’s staking pools according to each validator’s weight. Then, they are divided among delegators in proportion to each delegator’s stake.
+
+### What is the incentive to run a validator ?
+
+Validators earn additional rewards in the form of commissions on the fees that are collected. These fees can be collected not only in ATOMs but in any token that has been whitelisted by ATOM holders. 
+
+Validators also play a major role in governance. If a delegator does not vote, it inherits the vote from its validator. This gives validators a major responsibility in the ecosystem.
+
+### What is the probability of being chosen as a proposer for any given block? Does it go up with the quantity of ATOMs staked ?
+
+The validator that is selected to propose the next block is called proposer. The probability of being chosen is equal to the relative total stake (where total stake = self-bonded stake + delegators stake) of the validator. For example, if the total bonded stake across all validators is 100 ATOMs and a validator's total stake is 10 ATOMs, then this validator has a 10% probability to be chosen as the next proposer.
+
+### What is a commission?
+
+Fee rewards received by a validator’s pool are split between the validator and its delegators. The part that goes to the validator is called the validator’s commission. Each validator is free to set its initial commission, but there will be rules on how to change it.
+
+Note that validators cannot set commission on block rewards (also called inflationary rewards or provisions). This is to prevent a continuous shift in distribution from delegators to validators which would further the concentration of ATOMs and ultimately weaken the security of the system.
+
+### How are block rewards distributed?
+
+Block rewards are socially distributed, meaning that each block reward is distributed to each validator according to their stake.
```

Comment:
Block provisions are distributed proportionally to all validator relative to their existing stake. This means that even though each validator gains atoms with each provision, all validators will still maintain equal voting power.

Reply:
I think we should consider using weight instead of voting power for communications in the future, as voting power might be confused with governance

Reply:
consensus "weight" instead? I'm not opposed to the idea - maybe open an issue under `cosmos/cosmos`

Reply:
Did that